.goreleaser.yml

```diff
@@ -12,6 +12,12 @@ release:
     - glob: ./manifest/charts/*
     - glob: ./manifest/yamls/*
 
+variables:
+  fetcherDescription: Fetcher is a very lightweight component and all of related logic is in fetcher package itself. Fetcher helps in fetching and uploading code and in specializing environments.
+  builderDescription: Builder builds the source code into executable binary files.
```

Comment:
The builder assists in building the fission function source code for deployment.

Reply:
Done